capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -52,6 +52,12 @@ class ClubPage extends Component {
 
     this.setState({ club });
 
+    if (!club.memberIDs.includes(window.localStorage.getItem('userID')) 
+        && club.ownerID !== window.localStorage.getItem('userID')) {
+      alert('You are not a member of this club!');
```

Comment:
nit: This technically checks that you're neither an owner nor a member of the club, right?

Reply:
It does! In the case that you're neither a member nor the owner, you won't be able to see any club information. This prevents users from navigating directly to a clubpage link and seeing user comments or any other club information.